### PR TITLE
Centered alert message. Cleaned up headers

### LINKED ISSUE #169: Feedback email subject "Feedback for Vocable vVersion" has erroneous `v`
"Feedback for Vocable vVersion ..." should be changed to "Feedback for Vocable version ..."

### PR DESCRIPTION
**Pull request by stevefosterwta**
This PR also updates the project settings. 

image

### CODE REVIEW

File: Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift [hunk 1]
```diff
@@ -263,6 +264,8 @@ final class GazeableAlertViewController: UIViewController, UIViewControllerTrans
                 actionButtonStackView.addArrangedSubview(button)
             } else {
                 let separator = DividerView()
+                separator.translatesAutoresizingMaskIntoConstraints = false
+                separator.widthAnchor.constraint(equalToConstant: 1).isActive = true
```
Comment (stevefosterwta) [inline Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift]: There's has to be a better way...

File: Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift [hunk 2]
```diff
@@ -145,6 +145,7 @@ final class GazeableAlertViewController: UIViewController, UIViewControllerTrans
         label.textColor = .black
         label.lineBreakMode = .byWordWrapping
         label.numberOfLines = 0
+        label.textAlignment = .center
```
Comment (stevefosterwta) [inline Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift:155]: This matches designs.

File: Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift (hunk 1 above)
Comment (Clstroud) [reply Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift]: Maybe setting the content hugging priority to .required for the axis that matches the stack view axis?

File: Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift [hunk 3]
```diff
@@ -187,12 +188,12 @@ final class GazeableAlertViewController: UIViewController, UIViewControllerTrans
     }
 
     private func updateContentForCurrentTraitCollection() {
-        if traitCollection.horizontalSizeClass == .regular {
-            titleLabel.font = .systemFont(ofSize: 34)
-            titleContainerView.layoutMargins = UIEdgeInsets(top: 40, left: 50, bottom: 40, right: 50)
-        } else {
+        if [traitCollection.horizontalSizeClass, traitCollection.verticalSizeClass].contains(.compact) {
```
Comment (jsmorgan42) [inline Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift:198]: Love this syntax

File: Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift (hunk 1 above)
Comment (jsmorgan42) [reply Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift]: I think it's fine 🤷‍♂

**Review by jsmorgan42 — APPROVED** [review]
👨‍🎤 🎸

**Review by Clstroud — APPROVED** [review]
LGTM 👍

**Pull request merged**